src/lambda.rs

```diff
@@ -0,0 +1,68 @@
+use aws_sdk_lambda::error::InvokeAsyncError;
+use bytes::Bytes;
+
+use aws_sdk_lambda::model::LogType::Tail;
+use aws_sdk_lambda::output::InvokeAsyncOutput;
+use aws_sdk_lambda::types::{Blob, SdkError};
+use aws_sdk_lambda::RetryConfig;
+use aws_smithy_types::base64;
+use aws_types::SdkConfig;
+
+pub struct LambdaClient {
+    client: aws_sdk_lambda::Client,
+}
+
+pub struct LambdaResult {
+    base64_logs: String,
+    payload: Vec<u8>,
+}
+
+impl LambdaResult {
+    pub fn get_logs(&self) -> String {
+        String::from_utf8(base64::decode(&self.base64_logs).unwrap())
+            .unwrap()
+    }
+    pub fn get_payload(&self) -> String {
+        std::str::from_utf8(&self.payload).unwrap().to_string()
```

Comment:
Sorry I didn't notice that you could make a similar change than in `get_logs` here:
```suggestion
        String::from_utf8(&self.payload.clone()).unwrap()
```
Note that the `clone()` was hidden in the call to `to_string()` anyway.

Reply:
thanks for all suggestion